Content.Server/Climbing/ClimbSystem.cs

```diff
@@ -124,7 +124,7 @@ private void OnClimbableDragDrop(EntityUid uid, ClimbableComponent component, re
             BreakOnStun = true,
             RaiseOnUser = user == entityToMove,
             RaiseOnTarget = user != entityToMove,
-            RaiseOnUsed = false,
```

Comment:
The event should only ever be raised directed at a single entity, not multiple. So the other arguments need to be changed to false.